unitary/examples/quantum_chinese_chess/move.py

```diff
@@ -176,3 +176,57 @@ def effect(self, *objects) -> Iterator[cirq.Operation]:
         target_0.reset(source_0)
         source_0.reset()
         return iter(())
+
+
+class SplitJump(QuantumEffect):
+    """SplitJump from source_0 to target_0 and target_1. The only accepted (default) move_variant is
+    - BASIC.
```

Comment:
Probably some more detail in the docstring would be helpful.  This doesn't really tell you much.

Maybe explain which kinds of chinese chess moves this covers and how it is implemented?

Reply:
Thanks Doug. It's added.